content/docs/how-we-work/drupal/drupal-seo.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Drupal SEO
+weight: 7
+---
+
+# Drupal SEO
+
+We configure sites that are optimized for SEO”, which improves their visibility on search engines like Google.
+
+One of the key aspects of SEO is the use of specific modules that can enhance a site’s search engine performance. For instance, the [XML Sitemap](https://www.drupal.org/project/xmlsitemap) module creates a map of your site that helps search engines understand its structure and efficiently crawl through its pages. The [Google Analytics](https://www.drupal.org/project/google_analytics) module integrates our site with Google’s analytics tool, providing valuable insights into how visitors interact with our site. The [Metatag](https://www.drupal.org/project/metatag) module allows us to add meta tags to our site, which are snippets of text that describe a page’s content and can influence how your website is understood and displayed by search engines.
```

Comment:
@Yemaneberhan-Lemma Just thought to check when we say `One of the key aspects of SEO is the use of specific modules...`, is using specific modules are the aspects of SEO? To me it sounds like using modules are the features of SEO.